Configuration option for showing/hiding title bar in chat embed

0xi4o opened the pull request:
- Adds a `showTitle` option to the chat embed configuration (default: `true`). Setting it to `false` will hide the title bar in chat embed in both full-page and popup modes.
- Fixes the title bar sticking to the top of the page in full page mode because of the `fixed` tailwind class. Now it's `absolute` positioned regardless of mode.

Related: https://github.com/FlowiseAI/Flowise/issues/1314

HenryHengZJ commented:
@0xi4o were you able to replicate this issue?
![image](https://github.com/FlowiseAI/FlowiseChatEmbed/assets/26460777/3c472e6e-0807-4a4a-b4e2-b9aa8218e497)

I tried to, but didnt manage to do so. If you can share how to replicate, that'd be great 
and can you attach a screenshot for the fix as well?

0xi4o commented:
@HenryHengZJ I've added a sample repo where I was able to reproduce the issue. I've also updated the PR with the steps to reproduce and screenshots of the fix.

HenryHengZJ commented:
awesome! really appricate you setting up that repo so I can reproduce it. looks good!
